mmv1/products/compute/api.yaml

```diff
@@ -3697,6 +3698,7 @@ objects:
           - :AH
           - :SCTP
           - :ICMP
+          - :L3_DEFAULT
```

Comment:
Should we make the docs changes to `ports` / `allPorts` that come w/ `L3_DEFAULT` as well?

Reply:
The changes to the descriptions outlined in the user guide, specifically!

Reply:
Done